stdlib/distutils/dist.pyi

```diff
@@ -59,20 +59,20 @@ class Distribution:
     def __init__(self, attrs: Mapping[str, Any] | None = None) -> None: ...
     def get_option_dict(self, command: str) -> dict[str, tuple[str, str]]: ...
     def parse_config_files(self, filenames: Iterable[str] | None = None) -> None: ...
-    def get_command_obj(self, command: str, create: bool = ...) -> Command | None: ...
-    global_options: Incomplete
+    def get_command_obj(self, command: str, create: bool = True) -> Command | None: ...
```

Comment:
You'll have to add an allowlist entry for this method. Stubtest is, strictly speaking, correct that the default value is `1` rather than `True`. But if we change the default value to `1`, then we have to change the annotation to `int`. And we don't want to change the annotation here, because users Really Should be passing `bool`s to this parameter -- it only has a default value of `1` for historical reasons.

The stdlib allowlist is here: https://github.com/python/typeshed/blob/main/tests/stubtest_allowlists/py3_common.txt

`setuptool` has its own allowlist, which is here: https://github.com/python/typeshed/blob/main/stubs/setuptools/%40tests/stubtest_allowlist.txt

Reply:
We did a similar thing with some `tkinter` functions here, that had `1` instead of `True` as the default for historical reasons: #9637